src/main/java/com/elytradev/concrete/network/NetworkContext.java

```diff
@@ -100,7 +100,7 @@ public NetworkContext register(Class<? extends Message> clazz) {
 					if (f.getAnnotation(Optional.class) != null) {
 						optionalCount.add(clazz);
 					}
-					WireField<?> wf = new WireField<>(f);
+					WireField<?> wf = new WireField(f);
```

Comment:
At the time I made this change, I wasn't aware that diamond operators would still work with Java 6.